src/main/scala/com/lightbend/coursegentools/MasterAdm.scala

```diff
@@ -0,0 +1,76 @@
+package com.lightbend.coursegentools
+
+object MasterAdm {
+  def main(args: Array[String]): Unit = {
+
+    import Helpers._
+    import java.io.File
+    import sbt.{IO => sbtIO}
+
+    def changeExerciseNr(exerciseName: String, newNr: Int): String = {
+      val ExerciseNameSpec(pre, d, post) = exerciseName
+      f"$pre%s${newNr}%03d$post"
+    }
+
+
+    val cmdOptions = MasterAdmCmdLineOptParse.parse(args)
+    if (cmdOptions.isEmpty) System.exit(-1)
+    val MasterAdmCmdOption(masterRepo, multiJVM, renumber, renumberOffset, renumberDelta) = cmdOptions.get
+
+    val tmpDir = sbtIO.createUniqueDirectory(masterRepo)
+    val exercises: Vector[String] = getExerciseNames(masterRepo)
+    val ExerciseNameSpec(_, firstExerciseNr, _) = exercises.head
+    val (exercisesRenumbered: Vector[String], _) =
+      (exercises foldLeft(Vector.empty[String], renumberOffset)) {
```

Comment:
Yes, that would be possible too. The idea behind `renumberOffset` was to be able to add exercises in front of the first exercise. Of course, just using `renumberDelta` _and_ starting the first renumbered exercise at `renumberDelta` instead of 0 would sort of achieve the same result.